Tests/scs-test-runner.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+# vim: set ts=4 sw=4 et:
+#
+# scs-test-runner.py
+#
+# (c) Matthias Büchse <matthias.buechse@cloudandheat.com>
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import os
+import os.path
+import shutil
+import subprocess
+import sys
+import tempfile
+import time
+
+import click
+import tomli
+
+
+logger = logging.getLogger(__name__)
+MONITOR_URL = "https://compliance.sovereignit.cloud/"
+
+
+class Config:
+    def __init__(self):
+        self.cwd = os.path.abspath(os.path.dirname(sys.argv[0]) or os.getcwd())
+        self.scs_compliance_check = os.path.join(self.cwd, 'scs-compliance-check.py')
+        self.ssh_keygen = shutil.which('ssh-keygen')
+        self.curl = shutil.which('curl')
+        self.secrets = {}
+        self.presets = {}
+        self.scopes = {}
+        self.subjects = {}
+        self._auth_token = None
+
+    def load_toml(self, path):
+        self.cwd = os.path.abspath(os.path.dirname(path))
+        with open(path, "rb") as fileobj:
+            toml_dict = tomli.load(fileobj)
+        self.scopes.update(toml_dict.get('scopes', {}))
+        self.subjects.update(toml_dict.get('subjects', {}))
+        self.presets.update(toml_dict.get('presets', {}))
+        self.secrets.update(toml_dict.get('secrets', {}))
+
+    @property
+    def auth_token(self):
+        if self._auth_token is None:
+            pass
+        with open(self.abspath(self.secrets['tokenfile']), "r") as fileobj:
+            self._auth_token = fileobj.read().strip()
+        return self._auth_token
+
+    def get_subject_mapping(self, subject):
+        default_mapping = self.subjects.get('_', {}).get('mapping', {})
+        mapping = {key: value.format(subject=subject) for key, value in default_mapping.items()}
+        mapping.update(self.subjects.get(subject, {}).get('mapping', {}))
+        return mapping
+
+    def abspath(self, path):
+        return os.path.join(self.cwd, path)
+
+    def build_check_command(self, scope, subject, output):
+        cmd = [
+            sys.executable, self.scs_compliance_check, self.abspath(self.scopes[scope]['spec']),
+            '--debug', '-C', '-o', output, '-s', subject,
```

Comment:
Is it intentional to always pass `--debug` to the compliance test script even if it is disabled via `scs-test-runner.py --debug=false`?

Reply:
I would like to keep these two separated. But the question is pertinent...